mkt/search/tests/test_filters.py

```diff
@@ -65,6 +67,24 @@ def test_no_fuzzy_multi_word(self):
         qs_str = json.dumps(qs)
         ok_('fuzzy' not in qs_str)
 
+    @override_settings(ES_USE_PLUGINS=True)
+    def test_polish_analyzer(self):
+        """
+        Test that the polish analyzer is included correctly since it is an
+        exception to the rest b/c it is a plugin.
+        """
+        with self.activate(locale='pl'):
+            qs = self._filter(data={'q': u'próba', 'lang': 'pl'})
```

Comment:
oh, `lang` here isn't needed b/c of the locale activation that was needed above. I'll remove.